cli.js

```diff
@@ -1,60 +1,64 @@
 #!/usr/bin/env node
 
-var pipboylib = require('pipboylib')
-var relay = require('pipboylib/lib/relay')
+var pipboylib = require('pipboylib');
+var relay = require('pipboylib/lib/relay');
```

Comment:
I come from a C++ background and semicolons are *the way of my people*, though I am prepared to accept defeat and write w/o the semicolons if that's the majority decision :smiley: